### PR TITLE
Added tests to travis ci

### PR DESCRIPTION
**Pull request by amilajack**
Travis will not do anything without tests. I added some tests to ensure converage

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,7 +1,35 @@
 sudo: false
+
 language: node_js
+
 node_js:
-  - "stable"
   - "4"
-  - "0.12"
-  - "0.10"
```
Comment (heldr) [inline .travis.yml]: Nice commit!

But a lot of users still using Datauri over node 0.10, 0.12. Also, could you please change version 6 to "stable".

**Comment by amilajack** [comment]
Changed!

**Comment by heldr** [comment]
Could please help me adding node.js 0.10 and 0.12 + fixing tests?

Sorry, I'm very busy right now.

**Comment by amilajack** [comment]
Fixing tests as in fixing the library?

**Comment by heldr** [comment]
What do you mean about fixing the library?
https://travis-ci.org/heldr/datauri/builds (see master)

It's ok if you can't help now(you already did it with the PR 😄), but I can't merge into master since the test neither pass neither ran in old node versions. We can merge it in branch "new" (2.0) though and get it on master soon.

**Comment by amilajack** [comment]
I will try my best to make some additional PR's. Not sure how much I can help tho because I'm not to experienced in data/image manipulation/processing in JS.

**Comment by amilajack** [comment]
But be on the lookup for some decent PR's :)

**Comment by heldr** [comment]
Just please change this PR target to branch "new" and no worries 🍵

**Comment by amilajack** [comment]
To follow convention, should we rename 'new' to dev-master?

**Comment by heldr** [comment]
Sure thing! but not now 4am ET lol

**Comment by amilajack** [comment]
What does this line refer to?

https://github.com/heldr/datauri/blob/master/test/spec/datauri.spec.js#L8

**Comment by amilajack** [comment]
Also `const cli = cli_cmd;` in cli.spec.js

**Comment by amilajack** [comment]
Oh never mind! Got it

**Comment by amilajack** [comment]
@heldr finally got around to adding full eslint and ES6 support:
https://github.com/amilajack/datauri

But in the process of doing this, I broke some css tests. Wonder if you can look into this.

**Comment by heldr** [comment]
Closing this PR since Node.js maintained version are 4+

**Pull request closed without merging**